tests/integration_tests/charts/data/api_tests.py

```diff
@@ -444,11 +444,11 @@ def test_chart_data_dttm_filter(self):
         else:
             raise Exception("ds column not found")
 
+    @pytest.mark.usefixtures("load_birth_names_dashboard_with_slices")
```

Comment:
I'm really not sure how this test worked before without the inclusion of the fixture which ensures (hopefully) that the tests are idempotent.